prow/cmd/hmac/main.go

```diff
@@ -178,11 +178,72 @@ func main() {
 		hmacMapForRecovery:    map[string]github.HMACsForRepo{},
 	}
 
+	if err := c.handleInvitation(); err != nil {
+		logrus.WithError(err).Fatal("Error accepting invitations.")
+	}
+
 	if err := c.handleConfigUpdate(); err != nil {
 		logrus.WithError(err).Fatal("Error handling hmac config update.")
 	}
 }
 
+func (c *client) handleInvitation() error {
+	if !c.newHMACConfig.AutoAcceptInvitation {
+		logrus.Debug("Skip accepting github invitations as not configured.")
+		return nil
+	}
+	// Accept repos invitations first
+	repoIvs, err := c.githubHookClient.ListCurrentUserRepoInvitations()
+	if err != nil {
+		return err
+	}
+	for _, iv := range repoIvs {
+		if iv.Permission != github.Admin {
+			logrus.Errorf("invalid invitation from %s not accepted. Want: %v, got: %s",
```

Comment:
Super nit: it'll be more clear to state the error is due to permission
```suggestion
			logrus.Errorf("invalid invitation from %s is not accepted. Permission want: %v, got: %s",
```